notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -12041,6 +12041,105 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
+violence in the media [A point was hammered home about the seemingly deranged practice of traveling back in time to watch people die horribly in historical disasters was not so different from "tuning into the six o'clock news".]
+facing organized pursuit [Tom was on the run from the FBI throughout the better part of the film.]
+the thrill of danger [The titular "thrill seekers" were people traveling back in time to watch people die horribly in disasters for kicks.]
+law enforcement [The FBI agents Baker and Stanton were after Tom, who'd escaped from custody after being taken in for hijacking a passenger plane.]
+romantic love [The story built up to Tom and Elizabeth falling in love, and they shared a passionate kiss at the story's conclusion.]
+father and son [Tom worked to save his young son Kevin, who was living with Tom's ex-wife, from dying in a stadium fire.]
+
+:: Minor Themes
+first day on the job [Tom was shown on his first day at work at the National Inquisitor TV news station.]
+the ship Titanic [It was revealed that a man from the future had gone back in time to personally witness the sinking of the Titanic.]
+video gaming [Tom's son Kevin was more interested in playing his video game than he was in speaking with his dad, Tom, over the phone.]
+what it is like on a passenger flight [When Tom learned that the passenger flight he was on was destined to be involved in a mid-air collision, he took matters into his own hands to avert the impending disaster.]
+aircraft hijacking [Tom hijacked the passenger plan he was on to stop it from colliding with another plane in mid-air.]
+fusion power [It was revealed that Southern California was destroyed in a "fusion meltdown" in the year 2048.]
+decision making in the face of uncertainty [Tom and Elizabeth deliberated over this conundrum: Should they interfere in history by preventing a stadium fire that was destined to take the lives of 11,000 people at the risk doing so might cause something even more terrible to happen?]
+the calculus of human life [Tom and Elizabeth deliberated over this conundrum: Should they interfere in history by preventing a stadium fire that was destined to take the lives of 11,000 people at the risk doing so might cause something even more terrible to happen?]
+time traveler's dilemma [Should Tom and Elizabeth prevent a stadium fire that was destined to take the lives of 11,000 people at the risk doing so might cause something even more terrible to happen?]
+human interpersonal relationship [The FBI buddy agents Baker and Stanton were after Tom.]
+mother and son [Kevin's mother and step-father took him to see a hockey game at Copps Coliseum.]
+husband and wife [Kevin's mother and step-father were reluctant to let Kevin's father Tom back into Kevin's life.]
+stepfather and stepson [Kevin's mother and step-father took him to see a hockey game at Copps Coliseum.]
+sports [An eagerly anticipated ice hockey match was about to take place at Copps Coliseum.]
```

Comment:
We should make ice hockey a theme. I know Canada's not very good at it, but swallow your shame.

Reply:
Done and done. Separate pull request sent!